### PR TITLE
added kubesec

### PR DESCRIPTION
**Pull request by sanketg86**

### CODE REVIEW

File: vk-definitions.json [hunk 1]
```diff
@@ -192,6 +192,14 @@
                 "GithubOwner": "k14s",
                 "GithubRepo": "ytt",
                 "ReleaseName": "ytt-linux-amd64"
+            },
+	    {
+	        "Cmd": "kubesec",
+                "VersionArg": "version",
```
Comment (drzero42) [inline vk-definitions.json]: kubesec does not accept the argument `version`. It looks like it accepts `--version`.

File: vk-definitions.json [hunk 2]
```diff
@@ -192,6 +192,14 @@
...
+                "VersionRegexp": "v(.+?)\n",
```
Comment (drzero42) [inline vk-definitions.json]: Version output does not start with a v, so this should just be `(.+?)\n`.

**Review by drzero42 — COMMENTED** [review]
A good approach to test out adding a new definition is this:

1. Add definition
2. Test locally with `vk debug --definitions=vk-definitions.json kubesec`. This will test if the Github repo is correct, and if the grabbed download URL is valid.
3. Do an install: `vk install --definitions=vk-definitions.json kubesec`
4. Run debug again: `vk debug --definitions=vk-definitions.json kubesec`. This will test if the versions arg and regexp are correct.

**Comment by sanketg86** [comment]
sanket@cpm-sanket:~/git/new-infra/vk-definitions(feature/kubesec_tool)$ vk debug --definitions=vk-definitions.json kubesec
```
Debugging tool kubesec
Struct: &program.GithubDirectDownloadProgram{Command:program.Command{Path:"/home/sanket/.local/bin", Cmd:"kubesec", VersionArg:"version", VersionRegexp:"v(.+?)\n"}, GithubProgram:program.GithubProgram{GithubOwner:"shyiko", GithubRepo:"kubesec", ReleaseName:"kubesec-{VERSION}-linux-amd64", DownloadURL:"", PreRelease:false}}
Is installed: false
Latest version: 0.9.2
Download URL: https://github.com/shyiko/kubesec/releases/download/0.9.2/kubesec-0.9.2-linux-amd64
```

sanket@cpm-sanket:~/git/new-infra/vk-definitions(feature/kubesec_tool)$ vk install --definitions=vk-definitions.json kubesec
```
kubesec version 0.9.2 has been installed.
```
sanket@cpm-sanket:~/git/new-infra/vk-definitions(feature/kubesec_tool)$ kubesec --version
```
0.9.2
```

**Review by drzero42 — APPROVED** [review]
LGTM

**Pull request merged**